actions/user_actions.jsx

```diff
@@ -518,7 +498,7 @@ export async function resendVerification(email, success, error) {
 export async function loginById(userId, password, mfaToken, success, error) {
     const {data: ok, error: err} = await UserActions.loginById(userId, password, mfaToken)(dispatch, getState);
     if (ok && success) {
-        await store.dispatch(getLicenseConfig());
+        loadCurrentLocale();
```

Comment:
The license was already loaded here, so that part was unnecessary.

The `loadCurrentLocale` here is all that's left over from `loadMe` was moved here because the rest of that function was already done by `completeLogin` in mattermost-redux